src/events.js

```diff
@@ -1,18 +1,40 @@
+/**
+ * EventObserver is used by Syft, Job, and GridAPIClient to subscribe, unsubscribe or broadcast events.
```

Comment:
I think we don't need to list all classes that use EventObserver, at it's likely to change :)